libs/components/core/testing/src/help/help-testing.module.ts

```diff
@@ -15,5 +16,6 @@ import { SkyHelpTestingService } from './help-testing.service';
     },
     SkyHelpTestingController,
   ],
+  exports: [NoopAnimationsModule],
```

Comment:
Originally, I thought this module was for help inline (my mistake), but it's only for the help service's testing controller--so in this case we should not export the `NoopAnimationsModule` from this module because it's needed only for the `SkyPopoverComponent` (not the help service). You can put `NoopAnimationsModule` back in the file-drop spec's TestBed. Sorry!

(Ideally, we would have a `SkyFileAttachmentTestingModule` that would export `NoopAnimationsModule` for unit tests, but that's another story for another time.)

Reply:
reverted!